astropy/io/ascii/src/tokenizer.c

```diff
@@ -47,8 +47,8 @@ void delete_data(tokenizer_t *tokenizer)
     int i;
 
     if (tokenizer->output_cols)
-	for (i = 0; i < tokenizer->num_cols; ++i)
-	    free(tokenizer->output_cols[i]);
+    for (i = 0; i < tokenizer->num_cols; ++i)
+        free(tokenizer->output_cols[i]);
```

Comment:
I'll try to fix these up.  This is just after I did a find/replace of tabs for spaces in this file.  Depending on one's editor's tab width they previously *looked* okay, but only because the code was mixing tabs with spaces :(